mautic/mautic
# revert logic so correct template is included

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #12589 



#### Description:

the logic in selecting the correct template is wrong:
when there is no theme defined, the theme template is used (which doens't exist).
This PR corrects the logic, so the correct template is used



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. see #12589

be aware that you won't be able to save the focus item, as there is another bug, that will be tackled in another PR
